TAO/TAO_IDL/fe/idl.ypp

```diff
@@ -1476,14 +1525,14 @@ state_member
               AST_Decls &decls = *$2;
               for (size_t i = 0; i < decls.size (); i++)
                 {
-                  AST_Field * field = dynamic_cast<AST_Field*>(decls[i]);
+                  AST_Field * field = dynamic_cast<AST_Field*> (decls[i]);
```

Comment:
```suggestion
                  AST_Field *field = dynamic_cast<AST_Field *> (decls[i]);
```
...if we're going to change spacing

Reply:
Didn't notice that, I was specifically searching for `\S\(`.